### PR TITLE
added initial documentation and code snippets

### PR DESCRIPTION
**Pull request by djvoa12**
screen shot 2015-04-30 at 1 07 26 pm

### CODE REVIEW

**Comment by twokul** [comment]
I'm not 100% on this PR.

**Comment by pzuraq** [comment]
Thomas was having trouble getting a code snippet preprocessor to work. Definitely think we should use one though, any preferences?

**Comment by twokul** [comment]
@pzuraq https://github.com/Ticketfly/frontend-ui-components/pull/3

File: package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "ember-export-application-global": "^1.0.2",
     "ember-modal-dialog": "0.4.1",
     "ember-moment": "1.1.1",
-    "ember-popup-menu": "git://github.com/twokul/ember-popup-menu.git",
+    "ember-popup-menu": "0.1.19",
```
Comment (djvoa12) [inline package.json]: I made the change.  I think this PR can be merged as is.  We can make further style changes later when we got all component code and new logo?

@twokul

**Pull request closed without merging**